src/presence_analyzer/static/presence_start_end.html

```diff
@@ -90,39 +90,41 @@
                     dropdown.show();
                     loading.hide();
                 });
+
                 $('#user_id').change(function(){
                     var selected_user = $("#user_id").val();
                     var chart_div = $('#chart_div');
                     if(selected_user) {
                         loading.show();
                         chart_div.hide();
                         
-                        // some dummy data, this should go from API
-                        var result = [
-                            ["Mon", new Date(1, 1, 1, 7, 37, 38), new Date(1, 1, 1, 15, 07, 14)],
-                            ["Tue", new Date(1, 1, 1, 8, 16, 23), new Date(1, 1, 1, 14, 01, 47)],
-                            ["Wed", new Date(1, 1, 1, 7, 00, 06), new Date(1, 1, 1, 13, 24, 12)],
-                            ["Thu", new Date(1, 1, 1, 7, 26, 32), new Date(1, 1, 1, 12, 40, 45)],
-                            ["Fri", new Date(1, 1, 1, 6, 53, 22), new Date(1, 1, 1, 12, 52, 09)]
-                        ] 
+                        var result = [];
+
+						$.getJSON("/api/v1/presence_start_end/"+selected_user, function(tmp) {
+                            $.each(tmp, function(index, value) {
+                                result[index] = [value[0],
+                                new Date("1-1-1 " + String(value[1])),
```

Comment:
Value 1,2 It's already a string. 
Style.
Convert data which comes from the API, no need for another list.